File: Python3/first_unique_character_in_a_string_easy.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        hashmap = {}
        
        for i, w in enumerate(s):
            if w not in hashmap: 
                num = s.count(w)
                if num == 1: return i
                else: hashmap[w] = num
            else:
                pass

        
        return -1

File: C++/first_unique_character_in_a_string_easy.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        map<char, int> hashmap;
        
        for (char c : s) ++ hashmap[c];
        
        for (int i = 0; i < s.size(); ++i) {
            if(hashmap[s[i]] == 1) return i;
        }
        
        return -1;
    }
};
